src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/memories/JqlMemory.kt

```diff
@@ -3,5 +3,10 @@ package com.atlassian.performance.tools.jiraactions.api.memories
 import com.atlassian.performance.tools.jiraactions.api.page.IssuePage
 
 interface JqlMemory : Memory<String> {
+
     fun observe(issuePage: IssuePage)
+
+    fun recallByTag(filter: (String) -> Boolean): String? {
```

Comment:
This uses the Kotlin default implementation, which is not consumable by Java implementors, which breaks them, unless we use `@JvmDefault`.

Reply:
Ok